src/pybamm/parameters/parameter_values.py

```diff
@@ -319,115 +325,22 @@ def update(self, values, check_conflict=False, check_already_exists=True, path="
         # reset processed symbols
         self._processed_symbols = {}
 
-    def set_initial_stoichiometry_half_cell(
```

Comment:
looks great, thanks! can we keep the old methods and have them call `_set_initial_state` and raise a deprecation warning so that this isn't breaking?